### PR TITLE
refactor: include full attributes table for all enemies

### PR DESCRIPTION
**Pull request by Uberbagel**
bandits
barbarians
beasts
civilians
military
southern
undead
comment vanilla values

### CODE REVIEW

File: mod_reforged/hooks/config/faction_military.nut [hunk 1]
```diff
@@ -82,7 +191,7 @@
 	],
 	FatigueRecoveryRate = 15
 };
-::Const.Tactical.Actor.RF_ArbalesterHeavy <- {
+::Const.Tactical.Actor.RF_ArbalesterHeavy <- { // Arbelester
```
Comment (Darxo) [inline mod_reforged/hooks/config/faction_military.nut:194]: typo Should say Arb**a**lester

**Review by Darxo — APPROVED** [review]

**Comment by Uberbagel** [comment]
oop, sorry darxo, saw that you already approved.

**Review by Darxo — APPROVED** [review]

**Review by LordMidas — APPROVED** [review]

**Comment by LordMidas** [comment]
This necessitates that we remain vigilant over time to update the comments as per vanilla updates (if any future vanilla updates change these attributes of the entities).

**Comment by Uberbagel** [comment]
No problem there

**Pull request merged**